src/components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue

```diff
@@ -30,121 +30,24 @@
           {{ title }}
         </template>
 
-        <el-form
-          label-position="top"
-          size="small"
-          @submit.native.prevent="notSubmitForm"
-        >
-          <el-row>
-            <field-definition
-              v-for="(fieldAttributes) in storedFieldsListQuery"
-              :key="fieldAttributes.columnName"
-              :metadata-field="fieldAttributes"
-              :container-uuid="uuidForm"
-              :container-manager="containerManagerList"
-              :size-col="6"
-            />
-          </el-row>
-        </el-form>
+        <query-criteria
```

Comment:
Discard all changes on this file. The changes in this file have nothing to do with the scope of the pull reques